### DISCUSSION TITLE
TypeErrors when trying to plot time-series data

### DISCUSSION THREAD

**Discussion by rojasgabriel** [AdaptiveMotorControlLab/AmadeusGPT#6] [Q&A] [answered]
Hello! I started testing out the package and it looks very promising.

I'm unsure if this is the right forum for this or not, but the only feedback I have right now is that it struggles quite a bit when I ask it to plot data as a time-series. For context, I used the EPM demo and partitioned the ROIs so that I could have open vs. closed arms. I was trying to see if I could get the proportion of time spent in these two categories every 3 mins (or 5400 frames assuming 30fps). I tried it both ways with additional instructions and it seemed to always crash with a different kind of TypeError (calculating length of an int64, using an ".fps" property that didn't exist, etc.

**Comment by yeshaokai** [comment, marked as answer]
@rojasgabriel I am sorry to get back to you so late. We now release the code and you are welcome to try to git clone the repo (We are still working on pypl package, but soon). Many issues should have been solved with our newest code.
